source/Pages/Notification/components/Sign/components/Details/components/DisplayBox.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+import YellowInfo from '@assets/icons/yellow-info.svg';
+import useStyles from '../styles';
+
+const DisplayBox = ({
+  toggleModal, request, shouldWarn, title, subtitle, img,
+}) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  return (
+    <div className={classes.assetContainer}>
+      <div className={classes.assetText}>
+        <h2 className={`${classes.amountTitle} ${shouldWarn ? classes.yellowTitle : ''}`}>
+          {title || t('sign.warning.unknownArguments')}
+          {shouldWarn && (
+          <img
+            src={YellowInfo}
+            className={classes.yellowInfoIcon}
+            onClick={toggleModal}
+            alt="more-info"
+          />
+          )}
+        </h2>
+        <p className={classes.amountDescription}>
+          <span>{subtitle || '???'}&nbsp;</span>
+        </p>
+      </div>
+      <img src={img || request?.canisterIcon} className={classes.assetImg} alt="asset img" />
```

Comment:
no need for `img` text in `alt` - devtools will shout at us :D
just `asset` will be ok